src/main/java/barcus/command/Command.java

```diff
@@ -0,0 +1,14 @@
+package barcus.command;
+
+import barcus.tasklist.TaskList;
+import barcus.ui.Ui;
+import barcus.storage.Storage;
+import barcus.exception.BarcusException;
+
+/**
+ * Abstract command class
+ */
+public abstract class Command {
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws BarcusException;
+    public abstract boolean isExit();
```

Comment:
Many commands override this method to return false except for the ExitCommand class. Is it possible to make this method in the Command class concrete and override it to return true only for the ExitCommand class?
```suggestion
    public boolean isExit() {
        return false;
    }
```